### PR TITLE
_canShowModelAnswer added to the properties.schema

### PR DESCRIPTION
**Pull request by sarveshwar-gavhane**
Fixes https://github.com/adaptlearning/adapt_framework/issues/996

### CODE REVIEW

**Comment by sarveshwar-gavhane** [comment]
https://github.com/adaptlearning/adapt_framework/issues/996

**Comment by oliverfoster** [comment]
+1

**Comment by hrajotia** [comment]
+1

File: properties.schema [hunk 1]
```diff
@@ -37,6 +37,15 @@
...
       "help": "How many attempts the learner is allowed"
     },
+    "_canShowModelAnswer": {
+      "type":"boolean",
+      "required":true,
+      "default": true,
+      "title": "Display Model Answer",
+      "inputType": {"type": "Boolean", "options": [true, false]},
+      "validators": [],
+      "help": "Select 'true' to display the Model Answer"
```
Comment (moloko) [inline properties.schema]: could you change this to "Select 'true' to allow the user to view the 'model answer' should they answer the question incorrectly"?

**Comment by moloko** [comment]
see help text change, could you make the same change to the other PRs too? No need to get the +1's again since it's just a text change.

+1 for when that's done

**Comment by moloko** [comment]
+1

**Pull request merged**